traitsui/editors/text_editor.py

```diff
@@ -102,6 +102,9 @@ class ToolkitEditorFactory(EditorFactory):
     #: In a read-only text editor, allow selection and copying of the text.
     readonly_allow_selection = Bool(False)
 
+    #: Grayed-out placehold text to be displayed when the editor is empty.
+    text = Str()
```

Comment:
I have a moderate preference for `placeholder` to match https://github.com/enthought/pyface/blob/master/pyface/fields/i_text_field.py#L35

`text` is too generic - it could mean the actual text stored in the widget.